native-image-tests/build.gradle

```diff
@@ -69,6 +69,7 @@ graal {
   option "-H:+TraceClassInitialization"
   option "--initialize-at-build-time=kotlin.text.Charsets"
   option "-H:EnableURLProtocols=http,https"
+  option "-H:+AddAllCharsets"
```

Comment:
We should document which of these options our users need for OkHttp to behave correctly. For example, I suspect `TraceClassInitialization` is only for testing, but `kotlin.text.Charsets` is necessary for production? (and if it is, should it be?)

Reply:
Is there a way for us to promote `--initialize-at-build-time=kotlin.text.Charsets` into one of our JSON files? The more stuff that’s in those, the more stuff we can change in the future without breaking consumers.

Reply:
I think kotlin.text.Charsets is a bug, probably came out of work I'm doing in the AutomaticFeature.